clustered/integration-test/src/test/java/org/ehcache/clustered/LeaseTest.java

```diff
@@ -124,73 +120,30 @@ public void leaseExpiry() throws Exception {
     assertThat(cache.get(2L), equalTo("The two"));
     assertThat(cache.get(3L), equalTo("The three"));
 
-    setDelay(2000);
-    Thread.sleep(2000);
+    setDelay(6000, proxies);
+    Thread.sleep(6000);
     // We will now have lost the lease
 
-    setDelay(0L);
+    setDelay(0L, proxies);
 
-    assertThat(cache.get(1L), is(nullValue()));
-    assertThat(cache.get(2L), is(nullValue()));
-    assertThat(cache.get(3L), is(nullValue()));
+    CompletableFuture<String> future = CompletableFuture.supplyAsync(() -> {
+      while (true) {
+        try {
+          Thread.sleep(200);
+        } catch (InterruptedException e) {
+          //
```

Comment:
You probably shoud rethrow the interruption

Reply:
This is test code, would it really matter

Reply:
Yes it matters. Because you might get stuck in the loop if the test harness tries to interrupt you